symbolic-demangle/src/lib.rs

```diff
@@ -133,6 +136,10 @@ fn is_maybe_msvc(ident: &str) -> bool {
     ident.starts_with('?') || ident.starts_with("@?")
 }
 
+fn is_maybe_md5(ident: &str) -> bool {
+    ident.starts_with("??@") && ident.ends_with("@") && ident.len() == 36
```

Comment:
Hm, looks like I was missing that from my original PR, but it might be good to match that the chars in between are hexadecimal.

Reply:
If the 36 here is correct it needs a comment explaining why. This looks weird considering the prefix is variable length.

Reply:
There's a 3 character prefix, 32 hex characters, and a 1 character suffix. Unless I'm misunderstanding something?